pages/common/jpegoptim.md

```diff
@@ -0,0 +1,15 @@
+# jpegoptim
+
+> Optimise jpeg images.
+
+- Optimise a set of jpeg images:
+
+`jpegoptim {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Optimise jpeg images, stripping all non-essential information:
```

Comment:
It's unclear what is the difference between this example and the first. What does "non-essential information" mean? EXIF data? Something else?